### ISSUE TITLE
Awaiting Experiment : A-softmax loss

### ISSUE THREAD

**Issue by YiJingLin** [davidjaw/DLCV-final-Face-Recognition#6]
已完成A-softmax的串接([code-ref](https://github.com/pppoe/tensorflow-sphereface-asoftmax))
但硬碟bottleneck造成training需要 **9.x sec/step** , 請求算力支援實驗

## TODO ：
train In.-Res. with **A-softmax loss**

## Usage :
```
python train_teacher_a_softmax.py
```
